docs/package.json

```diff
@@ -84,7 +83,7 @@
     "marked": "^0.8.0",
     "material-table": "^1.50.0",
     "material-ui-popup-state": "^1.4.1",
-    "next": "8.1.0",
+    "next": "9.1.4",
```

Comment:
Upgrade to latest stable next release in a follow-up since they changed how they polyfill.